lms/djangoapps/save_for_later/api/v1/views.py

```diff
@@ -80,8 +83,6 @@ def post(self, request):
         )
         course_data = {
             'course': course,
-            'marketing_url': data.get('marketing_url'),
-            'org_img_url': data.get('org_img_url'),
```

Comment:
why are we removing this data?

Reply:
We are already passing request object to `send_email` helper function. we can do same thing in helper function that's why removing from here.
Like
`data = request.data` on line 37 in `/helper.py`